### PR TITLE
The New York Times - Main Page Clone Pull Request

### PR DESCRIPTION
**Pull request by jaspreet-singh-sahota**
> This project consist of making a clone of [this The New York Times article page](http://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0).

### The Milestones
- [x] Complete the *HTML* structure and *CSS* styles.
- [x] Complete the **index.html** validation by using [W3C Markup Validation Service](https://validator.w3.org/)
- [x] Resolve any issues found by the linter

_The project is complete and awaiting for approvement._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+- VSCode
+
+## Live Demo
+
+## Authors
+
+👤 **Jaspreet Singh**
+- Github: [@jaspreet-singh-sahota](https://github.com/jaspreet-singh-sahota)
+- Twitter: [@jaspree88033250]https://twitter.com/jaspree88033250
+- Linkedin: [jaspreet Singh]https://www.linkedin.com/in/jaspreet-singh-a28286146/
```
Comment (MauricioRobayo) [inline README.md:49]: There are parenthesis missing around the link 😄.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+    </head>
+    <body>
+        <script src="https://use.fontawesome.com/3b2ea06e46.js"></script>
+        <nav>
+            <div class="top-stripe-left">
+                <i class="fa fa-bars" aria-hidden="true"></i>
+                <i class="fa fa-search" aria-hidden="true"></i>
+                <span>SPACE & COSMOS</span>
+            </div>
+                <img class="the-newyork-times" src="https://www.de-simone.com/assets/new-york-times-logo-large-e1439227085840.jpg" alt="">
```
Comment (MauricioRobayo) [inline index.html]: The logo should be a link.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,552 @@
...
+                            pot expanding faster than light and cooling to an even temperature as it expands. That is something like
+                            inflation. And the structure of the universe mirrors the froth and foam of the original pot.</p>
+                    </div>
+                </div>
+            </div>
+            <p id="infographic-footer">By LARRY BUCHANAN and JONATHAN CORUM</p>
+        </div>        
+        <section id="grid-boxes">
+            <div class="more-in-space-container">
+                <h4>More in Space & Cosmos</h4>
```
Comment (MauricioRobayo) [inline index.html:346]: This should be **links** to other articles.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,552 @@
...
+                        <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/42/Distant_galaxy_GN-z11_in_GOODS-N_image_by_HST.jpg/300px-Distant_galaxy_GN-z11_in_GOODS-N_image_by_HST.jpg"  alt="space paths for women">
+                        <p class="link-pic-descr">Alison Farrand for the New York Times</p>
+                        <p class="link-pic-title">An Astronaut Who Built Paths to Space for Other Women</p>
+                        <p class="link-date">Oct. 2</p>
+                    </div>
+                </div>
+            </div>
+            <aside class="most-popular-container">
+                <h4>Most Popular</h4>
+                <p><a>Ja Morant Is the N.B.A.’s Newest Phenom</a></p>
```
Comment (MauricioRobayo) [inline index.html]: This should be links, please add the `href` attribute so the work as expected. You can use a `#` for the `href`: `<a href="#">Some article</a>`.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,552 @@
...
+                <p><a>Those People We Tried to Cancel? They’re All Hanging Out Together</a></p>
+                <p><a>Opinion: Now Comes the Naked Truth</a></p>
+                <p><a>In Trump’s Twitter Feed: Conspiracy-Mongers, Racists and Spies</a></p>
+                <p><a>Opinion: The NeverTrump Vindication</a></p>
+                <p><a>Opinion: The Real Reason Facebook Won’t Fact-Check Political Ads</a></p>
+                <p><a>Trump Takes In a Different Kind of Fight: U.F.C. in New York</a></p>
+                <p><a>Where ‘Jaws,’ the Ride, Lives Forever</a></p>
+                <p><a>Should I Tell the Children Why My Marriage Broke Up?</a></p>
+            </aside>
+            <div class="editors-picks-container">
```
Comment (MauricioRobayo) [inline index.html]: Please use an aside tag here for the Editor's picks.

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Hi! This looks awesome! Congratulations, this is good stuff.

Before continuing please make the following enhancements:

- [x] The NY Times logo on the navbar should be a link.
- [x] Image previews of articles at the bottom are links.
- [x] More popular articles on the rightmost side at the bottom are links.
- [x] The footer should have a fixed position (it sticks to the bottom part of the screen).
- [x] Editor's picks should use an `aside` tag.
- [x] Change the title of this PR and add a description:
image

**Status**: Request changes.

**Comment by nikasulo** [comment]
# @SlowKingV and @jaspreet-singh-sahota WOW! This is one of the best I've seen :open_mouth: 
Your project is really clean, no validation errors, great styling and effects! Also, great work implementing some of the changes listed above. You have just a few more things to correct and you may merge :medal_military:.

#### Subscribe section
You have structured this section masterfully, however, each element should be a link.

#### More in Space and Cosmos
This section was also beautifully written but each of the titles needs to be a link to the story, please fix this up! :smile: 

#### Editor's picks
Beautiful job! But please make each title a link and you are done!

#### Summary
I like your style team, you work great together obviously and you both have great potential. Please affect these few changes I listed above, submit for a re-review and you can be on your merry way :+1: 

##### If you require any help on this or subsequent projects, my messages are always open to chat, or if you'd just like to say hello, feel free all the same :smiley:. You can find me on [Twitter](https://twitter.com/oluwadamilareo_) and [Instagram](https://instagram.com/oluwadamilareolusakin). See you at the top :rocket:.

**Comment by shshamim63** [comment]
## Status :tada: [!! Approved !!] :tada:
___
Awesome job :clap:. The web-page looks almost exact as the actual one. Code is neat and clean. No violation of CSS is in there. Most important thing, linter is functional. well descriptive readme. In one word perfect. I got a suggestion to make. Let me know if you think it you have different thought on it.
___
### :palm_tree:  [Optional Changes]  :palm_tree:
___
You have some css codes on your style.css that are responsible for reset the browsers default styles. There are tons of reset code template you will get from online. So it means a lot of options. Suppose I want to contribute in your code and it makes me update my own choice of reset css. So first thing i have to find where is the reset css related code. Then i have to replace them. Don't you think it will be more fast accessible if you separate those codes in a different file called reset.css, A folder containing all the css file called style. I think it will make the code more organized.
___
#### This :+1: is from me. You can merge it and don't forget to fill the project completion form. Best of luck for your next project.
___
## :herb: Happy Coding :herb:
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](Shakhawat Hossain)

**Pull request merged**